tests/test-team/pages/routing/get-ready-to-move-page.ts

```diff
@@ -0,0 +1,18 @@
+import type { Locator, Page } from '@playwright/test';
+import { TemplateMgmtBasePage } from 'pages/template-mgmt-base-page';
+
+export class RoutingGetReadyToMovePage extends TemplateMgmtBasePage {
+  static readonly pathTemplate =
+    '/message-plans/get-ready-to-move/:messagePlanId';
```

Comment:
```suggestion
    '/message-plans/get-ready-to-move/:routingConfigId';
```
just for consistency with the actual route definition?